apps/ledger-live-mobile/src/families/cosmos/shared/02-SelectAmount.js

```diff
@@ -95,9 +95,17 @@ function DelegationAmount({ navigation, route }: Props) {
         ? validators.filter(v => !v.amount.eq(0))
         : validators;
 
-    const transaction = bridge.updateTransaction(tx, {
-      validators: filteredValidators,
-    });
+    const transaction = bridge.updateTransaction(
+      tx,
+      tx.mode === "delegate"
```

Comment:
No need to handle other modes? `undelegate`, `redelegate`?

Reply:
undelegate and redelagate go into the else case, amounts on those goes with the filteredValidators